find_max.rb

```diff
@@ -0,0 +1,19 @@
+
+# Find the maximum element between index 0 and max_index, exclusive
+
+def find_max(array, max_index)
+  max = nil
+  if max_index == 0
+    puts "Array is empty"
+  else
+    max = array[0]
+    index = 0
+    while index < max_index
+      if max < array[index]
```

Comment:
It might be more readable to put array[index] before max, because that's how we would normally say something like that ... i.e. "if x is greater than y", because we are looking for a number that is greater than.
```suggestion
      if array[index] > max
```